src/processing/SelectDelete.java

```diff
@@ -0,0 +1,50 @@
+package processing;
+
+import java.util.*;
+import processing.utility.*;
+import processing.boardobject.*;
+
+
+public class SelectDelete {
+    private static final BoardState board = ClientBoardState.maps;
+
+    public static ArrayList<Position> select (ArrayList <Position> positions) {
+
+        ObjectId mostProbableObjectId = board.getMostProbableObjectId(positions);
+        if (mostProbableObjectId == null) {
+            board.setSelectedObject(new PriorityQueueObject(null, null));
+            return new ArrayList <Position>();              // returning empty list
+        }
+
+        BoardObject selected = board.getBoardObjectFromId(mostProbableObjectId);
+        if (selected == null) {
+            board.setSelectedObject(new PriorityQueueObject(null, null));
+            return new ArrayList<Position>();              // returning empty list
+        }
+
+        board.setSelectedObject(new PriorityQueueObject(selected.getObjectId(), selected.getTimestamp()));
+        return selected.getPositions();
+    }
+
```

Comment:
Author can you comment select section as done in delete section.
Other than that looks good :-)

Reply:
the `TODO` line needs one function call too

Reply:
done